### PR TITLE
Removed allow credentials checkbox and updated the exclude credentials check box

### PR DESCRIPTION
**Pull request by weiilug**
This updates includes:

1. removed the allow credentials check box and the logic on collect options
2. added back the checkbox for each card
3. updated the new added card isChecked status dependent on the `exclude all credentials` checkbox

### CODE REVIEW

**Review by agektmr — CHANGES_REQUESTED** [review]
Made some comments.

File: src/templates/partials/sidebar.html [hunk 1]
```diff
@@ -6,11 +6,11 @@
     title="Toggles all credential's checkboxes. These checkboxes are used to determine whether to include in `excludeCredentials` or `allowCredentials`.">
     <mwc-checkbox id="exclude-all-credentials" checked></mwc-checkbox>
   </mwc-formfield>
-  <mwc-formfield
+  <!-- <mwc-formfield
```
[the remark this answers is not in the document]
Comment (weiilug) [reply src/templates/partials/sidebar.html]: This check box is recovered for the new revision.

File: src/public/scripts/main.ts [hunk 2]
```diff
@@ -450,15 +459,15 @@ const removeCredential = (credId: string) => async () => {
...
-// const onToggleCheckboxes = (e: any): void => {
-//   const checked = !e.target.checked;
-//   const cards = document.querySelectorAll<HTMLDivElement>('#credentials .mdc-card__primary-action');
-//   cards.forEach(card => {
-//     const checkbox = card.querySelector<Checkbox>('mwc-checkbox');
-//     if (checkbox) checkbox.checked = checked;
-//   });
-//   e.target.checked = checked;
-// }
+ const onToggleCheckboxes = (e: any): void => {
```
[the remark this answers is not in the document]
Comment (weiilug) [reply src/public/scripts/main.ts]: onExcludeAllCredentials() added

File: src/public/scripts/main.ts [hunk 3]
```diff
@@ -170,13 +173,13 @@ const collectOptions = (
...
   } else {
     // "Force empty `allowCredentials`"" is not checked
-    if (!$('#empty-allow-credentials').checked) {
-      // cards.forEach(card => {
-      //   const checkbox = card.querySelector<Checkbox>('mwc-checkbox');
-      //   if (checkbox?.checked) credentialsList.push(card.id);
-      // });
-      cards.forEach(card => credentialsList.push(card.id));
-    }
+    //if (!$('#empty-allow-credentials').checked) {
```
[the remark this answers is not in the document]
Comment (weiilug) [reply src/public/scripts/main.ts]: this checkbox is recovered.

**Review by agektmr — APPROVED** [review]
LGTM

**Pull request merged**